ember_debug/view-debug.js

```diff
@@ -464,6 +464,13 @@ export default EmberObject.extend(PortMixin, {
     rect.top = rect.top + window.scrollY;
     rect.left = rect.left + window.scrollX;
 
+    // in Chrome 61, rect has an extra toJSON property which will break
+    // $(div).css(rect)
+    // Error: Uncaught TypeError: Illegal invocation
+    if (rect.hasOwnProperty('toJSON')) {
+      delete rect.toJSON;
```

Comment:
For the sake of better future compatibility (in case any browser adds any properties/methods), let's instead build a new object with selected properties instead of using `$.extend`.

Something like:
```js
rect = {
  left: rect.left, 
  top: rect.top + window.scrollY, 
  right: rect.right + window.scrollX,
  bottom: rect.bottom,
  width: rect.width, 
  height: rect.height
};
```